tests/tensor/cartesian_tensor_test.py

```diff
@@ -0,0 +1,18 @@
+# pylint: disable=not-callable, no-member, invalid-name, line-too-long, wildcard-import, unused-wildcard-import, missing-docstring
```

Comment:
Add a test with a formula provided by the user

Reply:
done